SU2_CFD/include/numerics_structure.hpp

```diff
@@ -2071,34 +2071,64 @@ class CUpwLin_AdjTurb : public CNumerics {
 };
 
 /*!
- * \class CUpwSca_TurbSA
- * \brief Class for doing a scalar upwind solver for the Spalar-Allmaral turbulence model equations.
+ * \class CUpwSca_Abstract
```

Comment:
Makes a lot of sense to me. By the way, we are using a very basic linear upwind scheme based on the velocity. Have you ever tried to use a more elaborate upwind? (e.g. something similar to the upwind scheme that we use for the mass conservation equation like using Roe's averaged variables).